ticketingApp/style.css

```diff
@@ -0,0 +1,150 @@
+/* :root {
```

Comment:
As a general rule, try to remove commented code, from your code base. 
You can always find the old implementation in GIT.